Documentation/ceph-disaster-recovery.md

```diff
@@ -231,9 +237,97 @@ kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `mon.count`.
 
-# Adopt an existing Rook Ceph cluster into a new Kubernetes cluster
+## Restoring CRDs After Deletion
+
+When the Rook CRDs are deleted, the Rook operator will respond to the deletion event to attempt to clean up the cluster resources.
+If any data appears present in the cluster, Rook will refuse to allow the resources to be deleted since the operator will
+refuse to remove the finalizer on the CRs until the underlying data is deleted. For more details, see the
+[dependency design doc](https://github.com/rook/rook/blob/master/design/ceph/resource-dependencies.md).
+
+While it is good that the CRs will not be deleted and the underlying Ceph data and daemons continue to be
+available, the CRs will be stuck indefinitely in a `terminating` state in which the operator will not
+continue to reconcile. Upgrades will be blocked, further updates to the CRs are prevented, and so on.
```

Comment:
The operator *will* continue to reconcile during deletion so that dependent resources can be removed safely. This wouldn't be possible if a mon had to fail over or if the user needed to heal the ceph cluster while user data was being backed up.

Reply:
The operator will only reconcile related to deleting resources so I'll clarify that updates to the CR will not be applied.